modules/luxon.js

```diff
@@ -0,0 +1,8210 @@
+/* eslint-disable */
+const luxon = (function (exports) {
+  function _defineProperties(target, props) {
+    for (let i = 0; i < props.length; i++) {
+      const descriptor = props[i];
+      descriptor.enumerable = descriptor.enumerable || false;
+      descriptor.configurable = true;
+      if ('value' in descriptor) descriptor.writable = true;
+      Object.defineProperty(target, descriptor.key, descriptor);
+    }
+  }
```

Comment:
- [ ] Kindly make sure to use arrow functions, in your code I can see you haven't used the arrow syntax in your functions kindly redo the functions.

Reply:
@danmainah Thank you for the review, but I have a question on this one, as this function is a default luxon download file for ES6 changing the function inside the luxon file is quite a time taking as it has multiple lines of code.